src/main/java/core/basesyntax/model/Message.java

```diff
@@ -24,10 +22,9 @@
 public class Message {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    @Column(name = "id", nullable = false)
     private Long id;
     private String content;
 
-    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
+    @OneToMany(cascade = {CascadeType.REMOVE, CascadeType.PERSIST, CascadeType.MERGE})
```

Comment:
should we use  CascadeType.MERGE here?
Does it work without it?

Reply:
yep it works without it

Reply:
I wasn't sure about it :( added it just in case :(